packages/estatico-boilerplate/src/demo/modules/slideshow/slideshow.js

```diff
@@ -174,10 +174,13 @@ class SlideShow extends EstaticoModule {
     MediaQuery.addMQChangeListener(this.resize.bind(this), this.uuid);
   }
 
-  fetchSlides() {
+  async fetchSlides() {
```

Comment:
Why not write a module so developers don't have to this evrytime something like this:

```js
export default async function estaticoFetch(){
    if(!window.fetch){
        await import('whatwg-fetch');
    }
    return window.fetch(...arguments);
}
```

Reply:
We needed this merged, so would you be keen on opening another PR with this feature?